src/main/java/me/karakelley/tictactoe/GameLoop.java

```diff
@@ -6,51 +6,61 @@
 public class GameLoop {
 
   private UserInterface userInterface;
-  private HumanPlayer humanPlayer1;
-  private HumanPlayer humanPlayer2;
-  private HumanPlayer[] players;
+  private Player player1;
+  private Player player2;
+  private Player[] players;
 
-  public GameLoop(UserInterface userInterface, HumanPlayer player1, HumanPlayer player2) {
-    this.userInterface = userInterface;
-    this.humanPlayer1 = player1;
-    this.humanPlayer2 = player2;
-    this.players = setUpPlayerOrder();
+  public GameLoop(UserInterface humanUserInterface, Player player1, Player player2) {
+    this.userInterface = humanUserInterface;
+    this.player1 = player1;
+    this.player2 = player2;
+    this.players = new Player[]{player1, player2};
   }
 
   public void start(BoardState boardState, Game game) {
-    userInterface.displayMessage("Welcome to Tic Tac Toe!\n");
-    userInterface.userPrompt("Please press \"ENTER\" to continue\n", new EnterValidator());
+    userInterface.greetUser();
+    userInterface.displayMessage("Press ANY key to continue\n");
+    userInterface.waitForKeyPress();
     userInterface.displayBoard(boardState);
 
     play(boardState, game);
   }
 
   private void play(BoardState boardState, Game game) {
     while (!game.gameOver(boardState)) {
-      nextPlayersTurn(boardState, game);
-    }
-    if (game.tie(boardState)) {
-      userInterface.displayMessage("It's a tie!\n");
+      nextPlayersTurn(boardState);
     }
+    gameResults(game, boardState);
```

Comment:
The winning/tie results are not printed to the console for me for some reason. Can you look into it?